x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -419,12 +456,14 @@ const processTypeToCube: Record<ResolverProcessType, keyof NodeStyleMap> = {
 /**
  * A hook to bring Resolver theming information into components.
  */
-export const useResolverTheme = (): {
+export const useResolverTheme = (
```

Comment:
:grey_question: re my comment in chat, we could memoize this right?

Reply:
ya